Add public organization profile (#1)
testcontainers/.github: profile/README.md

or just about anything that can run in a Docker container.

## Implementations

Testcontainers is available for the following languages:

* [Testcontainers for .NET](https://dotnet.testcontainers.org/)
* [Testcontainers for Go](https://golang.testcontainers.org/)
* [Testcontainers for Java](https://java.testcontainers.org/)
* [Testcontainers for Node.js](https://node.testcontainers.org/)

### Community Projects

* [Testcontainers for Clojure](https://cljdoc.org/d/clj-test-containers/clj-test-containers/)
* [Testcontainers for Haskell](https://github.com/testcontainers/testcontainers-hs)
* [Testcontainers for Python](https://testcontainers-python.readthedocs.io/en/latest/)
* [Testcontainers for Ruby](https://github.com/testcontainers/testcontainers-ruby)
* [Testcontainers for Rust](https://docs.rs/testcontainers/latest/testcontainers/)
* [Testcontainers for Scala](https://github.com/testcontainers/testcontainers-scala/)

## Modules Catalog

Testcontainers offers modules with preconfigured implementations for popular services 
such as databases, message brokers, and more. 
Find the full list of modules in the [Testcontainers Module Catalog](https://testcontainers.com/modules/). 

> If you own a Testcontainers module implementation, please submit it to the catalog by creating a pull request to [this repository](https://github.com/testcontainers/community-module-registry).

Reviewer: ```suggestion
If you own a Testcontainers module implementation,
please submit it to the catalog by creating a pull request to [this repository](https://github.com/testcontainers/community-module-registry).
```

## Getting Started

To get started with Testcontainers, 
please refer to the [What is Testcontainers, and why should you use it?](https://testcontainers.com/guides/introducing-testcontainers/) guide. 
We recommend you to start with the [Getting Started](https://testcontainers.com/getting-started/) guide.

## How to start?

> * [Getting started with Testcontainers for .Net](https://testcontainers.com/guides/getting-started-with-testcontainers-for-dotnet/)

Reviewer: ```suggestion
* [Getting started with Testcontainers for .NET](https://testcontainers.com/guides/getting-started-with-testcontainers-for-dotnet/)
```

* [Getting started with Testcontainers for Go](https://testcontainers.com/guides/getting-started-with-testcontainers-for-go/)
* [Getting started with Testcontainers for Java](https://testcontainers.com/guides/getting-started-with-testcontainers-for-java/)
* [Getting started with Testcontainers for Node.js](https://testcontainers.com/guides/getting-started-with-testcontainers-for-nodejs/)

## Testcontainers Desktop

> The [Testcontainers Desktop](https://testcontainers.com/desktop/) is a free companion app for open source Testcontainers libraries that makes local development and testing with real dependencies simple.

Reviewer: ```suggestion
[Testcontainers Desktop](https://testcontainers.com/desktop/) is a free companion app for the open source Testcontainers libraries that enriches the Testcontainers user-experience makes local development and testing with real dependencies even simpler.
```
